### PR TITLE
Ensure consistent artifact `sha` (remove `bestzip` and bring back `archiver`)

### LINKED ISSUE #881: different sha256sum for zip file when no code changes
# This is a Bug Report

## Description

I run serverless package twice without any changes in code and config. The I got the two zip files, when I compare them hash and found the hash is different.

serverless itself DO set the access and creation timestamp to 1970.01.01, but this plugin not.

## Additional Data

* ***Serverless-Webpack Version you're using***: 
5.5.1
* ***Webpack version you're using***:
4.42.0
* ***Serverless Framework Version you're using***:
2.23.0
* ***Operating System***:
Linux or MacOS
* ***Stack Trace (if available)***:

### PR DESCRIPTION
**Pull request by j0k3r**
Replace #911

Closes #881

### CODE REVIEW

**Review by vicary — APPROVED** [review]
LGTM

**Comment by Dizolivemint** [comment]
just curious if there are any plans to add test coverage and merge @vicary @j0k3r

**Comment by j0k3r** [comment]
We tried to add e2e tests but we failed for now.

File: lib/packageModules.js [hunk 1]
```diff
@@ -26,7 +26,82 @@ function setArtifactPath(funcName, func, artifactPath) {
   }
 }
 
-function zip(directory, name) {
+/**
+ * Copy pasted from Serverless
+ *
+ * @see https://github.com/serverless/serverless/blob/63d54e1537e10ae63c171892edd886f6b81e83f6/lib/plugins/package/lib/zipService.js#L65
+ */
+function serverlessZip(args) {
```
Comment (miguel-a-calles-mba) [inline lib/packageModules.js:34]: It seems the code coverage dropped because there are no unit tests for this new function.

Comment (j0k3r) [reply lib/packageModules.js:34]: It's hard to test unit it, in fact, we would like to provide e2e tests which will more easy to cover that code.

**Pull request merged**